routes/web.php

```diff
@@ -16,7 +16,7 @@
 
 Auth::routes();
 
-Route::group(['prefix'=>'admin', 'as'=>'backend.'], function() {
+Route::group(['prefix'=>'admin', 'middleware'=>['can:isAdmin'] , 'as'=>'backend.'], function() {
```

Comment:
authorization with middleware 'can:isAdmin' => maybe will improve in the next step.